specification/containerservice/Fleet.Management/fleetmember.tsp

```diff
@@ -34,6 +34,16 @@ scalar ClusterResourceId
     }
   ]>;
 
+@added(Versions.v2025_03_01)
+@doc("Represents label object for a fleet member.")
+model FleetMemberLabel {
```

Comment:
If this data structure can be used by other resource types beyond fleet member, it should be called Label and moved to a common file.